src/system/ui/pages/post/index.js

```diff
@@ -16,6 +16,50 @@ export const Post = ( { location, match, history } ) => {
 	const [ passwordVisible, setPasswordVisible ] = useState( 'protected' === item.visibility )
 	const parentOptions = useParentOptions( item.type )
 	const wpRest = getWpRest()
+	const [ featureThumbnail, setfeatureThumbnail ] = useState( item.thumbnail )
+	const [ thumbData, setthumbData ] = useState( {} )
+	const [ hasUpdateimg, sethasUpdateimg ] = useState( false )
+	const [ removeThumbnail, setremoveThumbnail ] = useState( false )
+
+
+	const uploadFeatureImage = () => {
+		const customUploader = wp.media( {
+			title: 'Select an Image',
+			id: 'fl-asst-media-upload',
+			button: {
+				text: 'Use this image as Feature image'
```

Comment:
Can we change this to "Choose Featured Image"

Reply:
Yes I updated.